interfaces/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Does GPL allow to use another license in git subtree?

Reply:
Yes, but perhaps not the other way around

Reply:
The thing with licenses is that the derivative work may be _more_ restrictive than its components.

turbo-geth (GPLv3) is more restrictive than interfaces or rust-sentry (Apache-2.0) so it's OK.